authorizer.js

```diff
@@ -1,32 +1,19 @@
 const { isAuthorized, runIfDev } = require('./utils')
 
-exports.handler = async function(event) {
+exports.handler = function(event, context, callback) {
+
   const secret = process.env.JWT_SECRET
   if (!secret) {
     console.log('Error acquiring secret from env')
 
-    // We're ignoring this lint error because the Lambdas are configured to respond
-    // to a thrown string 'JWTSecretError'. For the moment we'll leave this as is
-    // but long-term we should reconfigure the lambda to respond to a thrown Error
-    // instead of an explicit literal string
-    // TODO: reconfigure lambda to respond to a thrown Error instead of literal 'JWTSecretError'
-
-    // eslint-disable-next-line no-throw-literal
-    throw 'JWTSecretError'
+    throw Error('JWTSecretError')
   }
 
   if (!isAuthorized(event.authorizationToken, secret)) {
-    // We're ignoring this lint error because the Lambdas are configured to respond
-    // to a thrown string 'Unauthorized'. For the moment we'll leave this as is
-    // but long-term we should reconfigure the lambda to respond to a thrown Error
-    // instead of an explicit literal string. The configuration
-    // TODO: reconfigure lambda to respond to a thrown Error instead of literal 'Unauthorized'
-
-    // eslint-disable-next-line no-throw-literal
-    throw 'Unauthorized'
+    callback("Unauthorized")
```

Comment:
Might still need the eslint disable line and explanation.

Reply:
done.